lib/msal-browser/src/cache/BrowserCacheManager.ts

```diff
@@ -124,6 +127,36 @@ export class BrowserCacheManager extends CacheManager {
         }
     }
 
+    /**
+     * Searches all cache entries for MSAL accounts and creates the account key map
+     * This is used to migrate users from older versions of MSAL which did not create the map.
+     * @returns 
+     */
+    private createAccountKeyMap(): void {
+        const accountKeys = this.getItem(StaticCacheKeys.ACCOUNT_KEYS);
+        if (accountKeys) {
+            // Account key map already exists, no need to iterate through cache
+            return;
+        }
+
+        const allKeys = this.browserStorage.getKeys();
```

Comment:
Add config to skip this step if the developer knows they don't have old versions in the wild that they care about.

Reply:
That's already part of this PR, please see line 47 :)